### PR TITLE
Added different values to show in the searchbar on the bottom

### PR DESCRIPTION
**Pull request by VolkerGoeschl**
This allows showing different metrics in the searchbar for spotting spots of interrest in longer logs easier.
The dropdown to select those is right beside the workspace selection. The chosen mode will survive picking a different log from a file but will get reset to average throttle when loading a new file.
The following modes have been implemented (so far) 
Average motor throttle: this is the old behavior
maximum Stick input: highest absolute value of (roll/pitch/yaw) (find snap-moves)
maximum Motor differential: difference of highest and lowest motor output (find fast moves/desyncs - thanks to Zoggbarr for the suggestion)

### CODE REVIEW

**Comment by nerdCopter** [comment]
nice work!

- drop down doesnt adjust properly when window-width is not large.
- maybe move it down to the graph area.
image

File: js/flightlog_index.js [hunk 1]
```diff
@@ -261,7 +290,16 @@ function FlightLogIndex(logData) {
...
                 }
             }
-            
+            if (sourceIndex.maxRC.length > 0) {
+                for (j = 0; j < sourceIndex.maxRC.length; j++) {
+                    resultIndex.maxRC[j] = sourceIndex.maxRC[j] * 20 - 1000;
+                }
+            }
+            if (sourceIndex.maxMotorDiff.length > 0) {
+                for (j = 0; j < sourceIndex.maxRC.length; j++) {
```
Comment (tbolin) [inline js/flightlog_index.js:299]:
```suggestion
                for (j = 0; j < sourceIndex.maxMotorDiff.length; j++) {
```

**Comment by nerdCopter** [comment]
@VolkerGoeschl , you can create a proper branch rather easily in order to create a proper PR:

from your personal master branch,
```
git fetch --all 
git checkout -b Seekbar_stats
git rebase upstream/master
```
then push to your repo, and create a new PR from `Seekbar_stats` branch.

afterward you should fix your personal master:
```
git checkout master
git reset --hard 363484f
git pull upstream master
```

always code in a branch, not master

File: js/flightlog_index.js (hunk 1 above)
Comment (RubberQuads) [reply js/flightlog_index.js:299]: Thanx - they should be the same length but it should still be proper copypasta!

**Comment by RubberQuads** [comment]
> nice work!
> 
> * drop down doesnt adjust properly when window-width is not large.
> * maybe move it down to the graph area.
>   Untitled

I would have loved to do this (I tried this first) but that white spot vanishes when you hide the legend. Maybe I'll move it right into the seekbar like it's done in the spectrographs once I have a better grasp of how the canvases work.

**Comment by nerdCopter** [comment]
closing this one in favor of #635

**Pull request closed without merging**